ui/client/README.md

```diff
@@ -1,28 +1,37 @@
 # Prerequisites
-To run make sure that you have the newest version of nodejs (currently it is 6.3.0) and npm (currently it is 3.10.3).
-If you have older versions follow the insruction on http://askubuntu.com/questions/594656/how-to-install-the-latest-versions-of-nodejs-and-npm-for-ubuntu-14-04-lts
+
+To run make sure that you have the newest version of nodejs (currently it is 6.3.0) and npm (currently it is 3.10.3). If you have older
+versions follow the insruction
+on http://askubuntu.com/questions/594656/how-to-install-the-latest-versions-of-nodejs-and-npm-for-ubuntu-14-04-lts
 After that you may need to cleanup npm global modules (result of `sudo npm -g list` should be blank).
 
 # Run
+
 ```
-npm ci
-npm start
+npm ci && npm start
 open http://localhost:3000
 ```
 
-#Run tests from console
+# Tests
+
+## Unit (jest) tests
+
 ```npm test```
 
-#Run tests from IntelliJ - Karma plugin needed
-- config file: ...ui/client/karma.conf.js
-- browsers to start: empty
-- node interpreter: np. usr/bin/node
-- karma package: ...ui/client/node_modules/karma
-- env variables: NODE_ENV=production
+### Run tests from IntelliJ
+
+`Jest` should work out of the box, just click green arrow.
+
+## E2E (cypress) tests
```

Comment:
Please add that you should fill cypress/fixtures/env.json
 based on cypress/fixtures/env.json.template before first run - it "almost" obvious, but... ;)